### PR TITLE
[W5.10] [W11-A1] Wayne Neo

### PR DESCRIPTION
**Pull request by wzneo**
As part of LO, added User Stories and User Cases

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -59,8 +59,13 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | edit a person | to update particulars
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:62]: you mean edit a person's details :stuck_out_tongue:

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -88,6 +93,30 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
++**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook requests for confirmation
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:100]: Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).

**Comment by okkhoy** [comment]
@AzuraAiR 
Some comments added. Please ack & close the PR after reading comments.

**Comment by wzneo** [comment]
Noted with thanks

**Pull request closed without merging**